AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/Connection/DataStoreConnectionScenario1Tests.swift

```diff
@@ -219,7 +219,6 @@ class DataStoreConnectionScenario1Tests: SyncEngineIntegrationTestBase {
         wait(for: [getProjectAfterDeleteCompleted], timeout: TestCommonConstants.networkTimeout)
     }
 
-    // TODO: This test will fail until https://github.com/aws-amplify/amplify-ios/pull/885 is merged in
     func testListProjectsByTeamID() throws {
```

Comment:
you also need to re-enable the test by right clicking on the test from the Test Explorer and click "Enable testListProjectsByTeamID"

Reply:
this test calls `startAmplifyAndWaitForSync()` so you have to rebase from main branch to get the DataStore.start after configuring or add it in yourself
```
Amplify.DataStore.start { result in
                    guard case .success = result else {
                        XCTFail("Failed to start")
                        return
                    }
                }
```

to run the test successfully

Reply:
this test doesn't seem to work at all without client side predicate matching

Reply:
What this mean? the test passes successfully

Reply:
I thought Project1 and Team1 does not have the appropriate list Project operation to filter by teamId